client/asset/btc/btc.go

```diff
@@ -362,7 +360,7 @@ type ExchangeWallet struct {
 	// 64-bit atomic variables first. See
 	// https://golang.org/pkg/sync/atomic/#pkg-note-BUG
 	tipAtConnect      int64
-	node              *rpcClient
+	node              Wallet
```

Comment:
Without seeing how the spv btwallet integration works, I'm not entirely getting the motivation for this `Wallet` interface, which is a bunch of unexported methods, so it would have to be coming from this package anyway.  The interface makes code navigation just that much more difficult cause it won't just jump to an actual implementation.

I assume you're planning a different concrete type like `spvRPCClient` that embeds an `*rpcClient` and overrides certain methods?

I can live with this, but from a clarity point of view I'm just wondering if having a single concrete implementation that has a flag for spv mode to switch between method implementations would make for easier navigation... but you've gone further down this path so I defer to you.

Reply:
What you've described might make sense for the initial Decred SPV implementation that still uses an RPC client underneath. For Bitcoin SPV, we are working directly with the `*btcwallet.Wallet`. There is no RPC. The implementations have little in common.

Reply:
Makes sense. Thanks.